.gitignore

```diff
@@ -41,6 +41,5 @@ onnxprofile_profile_test_*.json
 /csharp/src/Microsoft.ML.OnnxRuntime/Microsoft.ML.OnnxRuntime.props
 cmake/external/FeaturizersLibrary/
 # Java specific ignores
-java/src/main/native/ai_onnxruntime_*.h
```

Comment:
This ignore line should come back now the JNI headers aren't checked in.

Reply:
I don't think it is needed, since the headers are generated in the java/build/headers directory and which is already ignored.

Reply:
Ok.

Reply:
The CMake file still looks for them in `java/src/main/native` on line 42?

Reply:
Line 42 refers to any manually created files like `OrtJniUtil.h`. If somebody wanted to add another util or helper object then it would automatically be picked up by CMake and git.

The generated headers are referred to at the end of line 51:
```
target_include_directories(onnxruntime4j_jni PRIVATE ${REPO_ROOT}/include ${JAVA_ROOT}/build/headers)
```